akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -56,7 +56,7 @@ sealed abstract class MediaType extends jm.MediaType with LazyValueBytesRenderab
       case _            ⇒ false
     }
 
-  override def hashCode(): Int = value.hashCode
+  override def hashCode(): Int = value.toLowerCase.hashCode
```

Comment:
val

Reply:
Nah, def is fine. No reason to increase the weight of the object for this